Sources/WebAPI/Methods/ChatPermalink.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public struct ChatPermalink: WebAPIRequest {
+    public let channelID: String?
+    public let ts: String
+    
+    public let endpoint = "chat.getPermalink"
+    public var body: [String : Any?] {
+        return ["channel": channelID, "message_ts": ts]
+    }
+    
+    public init(message: Message) {
+        self.channelID = message.channel?.id
+        self.ts = message.ts
+    }
+    
+    public func handle(response: NetworkResponse) throws -> String {
```

Comment:
It wasn't immediately clear to me whether it'd be preferable to have this as a `String` or ` `URL` – I leaned URL, but the direction of the project seemed to be "match the types Slack uses to represent fields internally," so I went `String`. If anyone thinks `URL` would be better, LMK and I'll change.